tests/test-notebook/src/actions.spec.ts

```diff
@@ -968,6 +974,11 @@ describe('@jupyterlab/notebook', () => {
         expect(widget.isSelected(widget.widgets[1])).to.equal(true);
       });
 
+      it('should extend the selection the bottomost cell', () => {
+        NotebookActions.extendSelectionBelow(widget, true);
+        expect(widget.isSelected(widget.widgets[0])).to.equal(true);
+        expect(widget.isSelected(widget.widgets[1])).to.equal(true);
```

Comment:
This only checks to see if the cell below the current is selected. Can you check all the way to the last cell to make sure each is selected?